src/api/posts.js

```diff
@@ -0,0 +1,9 @@
+import { BASE_URL } from './api';
+
+export const getUserPosts = () => fetch(`${BASE_URL}posts/`)
+  .then(response => response.json())
+  .then(result => result.data);
```

Comment:
```suggestion
export const getAllPosts = () => fetch(`${BASE_URL}posts/`)
  .then(response => response.json())
  .then(result => result.data);
```
you're not filtering this posts by userId, so it's rather getPost or getAllPosts

Reply:
+